addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -675,18 +675,19 @@ function enforceImagesResponsivity(editable) {
  * will be computed for the editable element's owner document.
  *
  * @param {JQuery} $editable
- * @param {Object[]} [cssRules] Array<{selector: string;
+ * @param {Object} options {$iframe: JQuery;
+ *                          cssRules: Array<{selector: string;
  *                                   style: {[styleName]: string};
  *                                   specificity: number;}>
- * @param {JQuery} [$iframe] the iframe containing the editable, if any
+ *                          wysiwyg: Object}
  */
-export async function toInline($editable, cssRules, $iframe) {
+export async function toInline($editable, options) {
     $editable.removeClass('odoo-editor-editable');
     const editable = $editable.get(0);
-    const iframe = $iframe && $iframe.get(0);
-    const wysiwyg = $editable.data('wysiwyg');
+    const iframe = options.$iframe && options.$iframe.get(0);
+    const wysiwyg = $editable.data('wysiwyg') || options.wysiwyg;
     const doc = editable.ownerDocument;
-    cssRules = cssRules || wysiwyg && wysiwyg._rulesCache;
+    let cssRules = options.cssRules || wysiwyg && wysiwyg._rulesCache;
```

Comment:
Do we need the wysiwyg for anything else than the rules cache? Why not pass `wywisyg._rulesCache` at call site into the `cssRules` option instead?

Reply:
Indeed we need it for `_rulesCache`  , but we need `wysiwyg` because we assign `_rulesCache` later in case we recalculate the `cssRules`.

Reply:
Makes sense. But then where do we use the `cssRules` option?

Reply:
I think it's just added to make the function more generic , so if you want to apply some custom `cssRules` and not the ones defined in `wysiwyg` you can do that too , otherwise in our usecase it's always undefined and we use `wysiwyg._rulesCache`

Reply:
I see what you mean and it makes sense, but in practice I feel like it's just a trap waiting for someone to fall for it because if some future code passes `wysiwyg._rulesCache` to it instead of `wysiwyg` and the rules have not been computed yet, then the cache will never be set and the rules will be recomputed each time. Do you see what I mean?

Reply:
I understand what you mean , makes sense, because people can just ignore wysiwyg then, I will remove it from the options and make only `wysiwyg` responsible for the `cssRules` , should be fine like that right ?

Reply:
Yes, thanks!